Release notes: Cluster-internal connectivity checks (#256)
arangodb/docs-hugo: site/content/3.11/release-notes/version-3.11/whats-new-in-3-11.md

if other databases are dropped first. The default value for this option is
unlimited, so an arbitrary amount of databases can be created.

### Cluster-internal connectivity checks

<small>Introduced in: v3.11.5</small>

This feature makes Coordinators and DB-Servers in a cluster periodically send

> check requests to each other, in order to check if all servers can connect to

Reviewer: Grammarly marks "to check if" as a problem 🤔 
Servers as a term is a bit ambiguous, I think nodes is better in this context.
```suggestion
check requests to each other, in order to see if all nodes can connect to
```

each other.
If a cluster-internal connection to another Coordinator or DB-Server cannot
be established within 10 seconds, a warning is now logged.

The new `--cluster.connectivity-check-interval` startup option can be used
to control the frequency of the connectivity check, in seconds.
If set to a value greater than zero, the initial connectivity check is
performed approximately 15 seconds after the instance start, and subsequent
connectivity checks are executed with the specified frequency.

> If set to a value of zero, connectivity checks are disabled.

Reviewer: ```suggestion
If set to `0`, connectivity checks are disabled.
```
